uint64 id
float64[] position
float64 p
float64 q
CLiFFMapDistribution[] distributions